src/app/pages/sharing/iscsi/target/delete-target-dialog/delete-target-dialog.component.html

```diff
@@ -0,0 +1,45 @@
+<h1 mat-dialog-title>{{ 'Delete Target "{name}"' | translate:{ name: target.name } }}</h1>
+
+<div mat-dialog-content>
+  @if (warningMessage) {
+    <p [innerHTML]="warningMessage"></p>
```

Comment:
Let's change the message to just `Warning: iSCSI target is currently in use` and make it yellow or orange, so that user does not think it's blocking.